x-pack/plugin/core/src/main/resources/monitoring-es.json

```diff
@@ -991,7 +991,15 @@
                   "type": "integer"
                 },
                 "exception": {
-                  "type": "text"
+                  "type": "nested",
```

Comment:
Hey @martijnvg, I was actually wondering if `fetch_exceptions` (not `exception`) should be `nested` as it contains an array of objects. Thoughts?

Reply:
@ycombinator I pushed: 1797b73aba2b2e5f7edea503b47ae2b4df8299da